### PR TITLE
[keymgr/dv] Sign off V2

### PR DESCRIPTION
**Pull request by weicaiyang**
depends on #8318 and #8247
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

File: hw/ip/keymgr/doc/checklist.md [hunk 1]
```diff
@@ -165,27 +165,27 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
...
+Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Done        |
+Documentation | [DV_DOC_COMPLETED][]                    | Done        |
+Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Done        |
+Testbench     | [ALL_INTERFACES_EXERCISED][]            | Done        |
+Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Done        |
+Testbench     | [SIM_TB_ENV_COMPLETED][]                | Done        |
+Tests         | [SIM_ALL_TESTS_PASSING][]               | Done        |
+Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | N/A         |
+Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | N/A         |
 Tests         | [SIM_FW_SIMULATED][]                    | Not Started |
```
Comment (weicaiyang) [inline hw/ip/keymgr/doc/checklist.md:177]: @sriyerg 
What should we do for this item? The description below is not very clear to me. Do you need to create a keymgr_smoketest in order to mark this done?

> Chip-level tests exist to verify example firmware code (DIFs) in simulation.

File: hw/ip/keymgr/data/keymgr.prj.hjson [hunk 2]
```diff
@@ -11,8 +11,8 @@
       {
         version:            "0.1",
         life_stage:         "L1",
-        design_stage:       "D1",
-        verification_stage: "V1",
+        design_stage:       "D2",
```
Comment (imphil) [inline hw/ip/keymgr/data/keymgr.prj.hjson:14]: Please don't do two stage transitions in the same PR. The design one should be separate, as it needs a different review.

Comment (weicaiyang) [reply hw/ip/keymgr/data/keymgr.prj.hjson:14]: Good point. Let me make another PR for that.

**Review by matutem — APPROVED** [review]
Great

**Review by sriyerg — APPROVED** [review]

**Review by cindychip — APPROVED** [review]
awesome! congrats

**Review by tjaychen — APPROVED** [review]
lgtm

**Comment by tjaychen** [comment]
@weicaiyang i know we traded some discussions how to prove a known value, i think we can discuss that separately. Maybe there are things we can simplify on the design side to make that happen.

**Comment by weicaiyang** [comment]
> @weicaiyang i know we traded some discussions how to prove a known value, i think we can discuss that separately. Maybe there are things we can simplify on the design side to make that happen.

sg, filed an issue #8458 to continue the discussion.

**Pull request merged**